core/src/main/java/org/jdbi/v3/core/config/ConfigRegistry.java

```diff
@@ -30,7 +35,11 @@
      * Creates a new config registry.
      */
     public ConfigRegistry() {
-        configs.put(JdbiCaches.class, new JdbiCaches());
+        get(JdbiCaches.class);
```

Comment:
I'd avoid if possible calling a non-private method from a constructor of a non-final class